CUSTOMIZATION.md

```diff
@@ -80,3 +83,10 @@ The header may optionally define the following:
 
 *   `etherpad` is the URL for the Etherpad for your workshop.  If you are
     not using an Etherpad, you can delete this value.
+
+## Home Page: Schedule, Syllabus, and Setup
+
+You should edit the sections titled `Schedule` and `Syllabus`
```

Comment:
I also like the commenting in index.html.

Would it be easier for the user to scan the file visually for the appropriate code block to edit if the comments included a section name? For example, adding a capitalized section title:
![screen shot 2015-01-21 at 05 17 00](https://cloud.githubusercontent.com/assets/2680980/5836637/d724b220-a12c-11e4-84db-a5c496365e0d.png)